tests/requirements-libraries.txt

```diff
@@ -63,6 +62,10 @@ numpy<=1.19.0; python_version == '3.5'  # pyup: ignore
 scipy==1.5.3; python_version > '3.5'
 scipy<=1.5.0; python_version == '3.5'  # pyup: ignore
 
+# Pillow 8.0 dropped support for python 3.5
+Pillow==8.0.0; python_version > '3.5'
+Pillow<8.0.0; python version == '3.5' # pyup: ignore
```

Comment:
There's an underscore missing in the bottom line (it was in the original commit as well).

Reply:
Thanks for noticing, it is corrected now.